jmeter/src/main/java/com/baeldung/dashboard/controllers/DashBoard.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.dashboard.controllers;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import com.baeldung.dashboard.models.Quotes;
+
+@Controller
+public class DashBoard {
+    
+    @GetMapping("/greeting")
+    public String getGreeting(Model model) {
+        model.addAttribute("host", System.getProperty("os.name"));
+        return "greeting";
+    }
+    
+    @GetMapping("/quote")
+    public String getQuote(Model model) throws InterruptedException {
+        Random r = new Random();
+        int day = r.nextInt(7);
+        String quote = Quotes.list.get(day);
+        
+        int wait = r.nextInt(6);
+        Thread.currentThread().sleep(wait);
+        model.addAttribute("quote", quote);
+        
+        return "quote";
+    }
+    
+    @GetMapping("/time")
+    public String getTime(Model model) throws InterruptedException {
```

Comment:
why not use: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/ModelAndView.html

```suggestion
    public String getTime(ModelAndView model) throws InterruptedException {
```

Reply:
Its a matter of preference. Right now its just a demo app to create some endpoints. I consider use of Model ok here. Thanks.